features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/DeveloperSettingsPresenter.kt

```diff
@@ -124,16 +124,12 @@ class DeveloperSettingsPresenter @Inject constructor(
         enabledFeatures: SnapshotStateMap<String, Boolean>,
         featureUiModel: FeatureUiModel,
         enabled: Boolean,
-        triggerClearCache: () -> Unit,
+        @Suppress("UNUSED_PARAMETER") triggerClearCache: () -> Unit,
```

Comment:
I prefer to keep this param, can be helpful for future usage.

Reply:
I had to add some exceptions to the kover verification since we're not testing this, but it should be fine now.